ql/src/test/results/clientpositive/llap/enforce_constraint_notnull.q.out

```diff
@@ -6411,53 +6364,16 @@ STAGE PLANS:
                     expressions: UDFToInteger(key) (type: int)
                     outputColumnNames: _col0
                     Statistics: Num rows: 500 Data size: 2000 Basic stats: COMPLETE Column stats: COMPLETE
-                    Filter Operator
-                      predicate: enforce_constraint(_col0 is not null) (type: boolean)
```

Comment:
Does this mean the not null constraint is not applied after the change?

Reply:
Yes, it means that not null constraint does not have any meaning for temporary tables. Before this patch, a temp table with constraints was getting persisted in metastore and therefore had constraint information persisted. But  after this patch, constraints information will get lost for temporary tables unless we store constraints related info in memory. 

@nrg4878 : Is the behavior for temporary table in terms on constraints as expected after this patch?